addon/initializers/m3-store.js

```diff
@@ -51,11 +51,23 @@ export function extendStore(Store) {
       return this._super(modelName);
     },
 
-    createModelDataFor(modelName, id, clientId, storeWrapper) {
+    createModelDataFor() {
+      let ret = this.createRecordDataFor(...arguments);
+
+      // fallback to older ember-data methodName super if needed
+      if (ret === undefined) {
+        return this._super(...arguments);
+      }
+
+      return ret;
+    },
```

Comment:
I'm not a fan of adding this complexity at runtime.

What are the versions involved here?

`createModelDataFor` is 3.4.x and `createRecordDataFor` is 3.5.0 right?

can we split these up with https://github.com/ember-cli/ember-cli-version-checker ?

Reply:
done